dattri/algorithm/base.py

```diff
@@ -117,104 +109,135 @@ def _set_full_train_data(self, dataloader: torch.utils.data.DataLoader) -> None:
         Args:
             dataloader (DataLoader): The dataloader for full training samples.
         """
-        # This function may be overrided by the subclass
+        # This function may be overridden by the subclass
         self.full_train_dataloader = dataloader
 
-    def generate_test_query(
+    def generate_test_rep(
         self,
-        index: int,
+        ckpt_idx: int,
```

Comment:
I think this should be OK? Since no one (even the TDA developers) need to explicitly write any code calling `generate_test_rep` / `transform_test_rep` / `generate_train_rep` / `transform_train_rep`. Then I guess this is okay to leave it as it is.